tests/pdf_viewer/test_pdf_download.py

```diff
@@ -0,0 +1,60 @@
+import os
+import platform
+import time
+
+import pytest
+from pynput.keyboard import Controller, Key
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+
+from modules.browser_object_navigation import Navigation
+from modules.page_object import GenericPdf
+
+
+@pytest.fixture()
+def add_prefs():
+    return []
+
+
+def test_pdf_download(driver: Firefox, fillable_pdf_url: str):
+    """
+    C3932: PDF files can be successfully downloaded via pdf.js
+    """
+    GenericPdf(driver, pdf_url=fillable_pdf_url).open()
+    keyboard = Controller()
+
+    # Click the download button
+    download_button = driver.find_element(By.ID, "download")
+    download_button.click()
+
+    # Allow time for the download dialog m to appear and pressing enter to download
+    time.sleep(2)
+    keyboard.press(Key.enter)
+    keyboard.release(Key.enter)
+
+    # Allow time for the download to complete
+    time.sleep(2)
+
+    # Determine the platform and set the expected download path
+    this_platform = platform.system()
+    saved_pdf_location = ""
+    file_name = "i-9.pdf"
+
+    if this_platform == "Windows":
```

Comment:
Likewise, you can remove all this code if you request the fixture...I basically copied this code and put it in conftest.

Reply:
kept file name.